config/PropertyNames.xml

```diff
@@ -301,64 +301,64 @@ generated from the section label.
     <class name="mx" prefix-only="false">
         <suffix name="GroupBy" />
         <suffix name="Map" />
-        <suffix name="RetainDetached" />
+        <suffix name="RetainDetached" default="10" />
         <suffix name="Accept" />
         <suffix name="Reject" />
     </class>
 
     <section name="Ice">
```

Comment:
There is no default for the ThreadPool properties above?

Reply:
There are, I just haven't gotten to them yet. IIRC they don't read them "regularly". Will look closer at the ones that are currently unset in a follow-up PR.